packages/editor-ui/src/mixins/pushConnection.ts

```diff
@@ -291,6 +291,30 @@ export const pushConnection = defineComponent({
 				}
 			}
 
+			if (receivedData.type === 'workflowFailedToActivate') {
+				this.workflowsStore.setWorkflowInactive(receivedData.data.workflowId);
+				this.workflowsStore.setActive(false);
+
+				this.showError(
```

Comment:
We should add an if to check whether the currently visible workflow ID matches the one we just received. The way this is currently working, I tested with 3 main instances and if I click the toggle in one of the followers, the other displays the error even if another totally unrelated workflow is open.

Reply:
Please see [a3147a9](https://github.com/n8n-io/n8n/pull/7643/commits/a3147a926a9501bf373b0bf5a63717f17fca97e8)